syntax = "proto3";

import "vlan_network_setting.proto";

message UpdateNetworkRequest {
  string id = 1;

  oneof type_specific_setting {
    VlanNetworkSetting vlan_network_setting = 2;
  }
}
